src/components/product/Product.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import './product.scss';
+
+class Product extends Component {
+
+    constructor(props) {
+
+        super(props);
+
+        this.fields = [
+            { label: 'title', reg: /^.{3,}$/ },
+            { label: 'price', reg: /^[0-9]+$/ },
+            { label: 'description', reg: /^.{3,}$/ },
+        ];
+
+        this.state = {};
+
+        this.fields.forEach(({ label }) => this.state[label] = ({
+            value: '',
+            error: false,
+            display: false,
+        }));
+    }
+
+    validate = ({ target }) => {
+        const field = this.fields.find(item => item.label === target.name);
+        const stateField = this.state[target.name];
+
+        !field.reg.test(target.value) ?
```

Comment:
please, do not do actions using ternary operator. In this case use `if ... else` (read [this](http://learn.javascript.ru/ifelse#netraditsionnoe-ispolzovanie))